src/dswx_sar/dummy_dswx_s1.py

```diff
@@ -1,20 +1,14 @@
 import os
```

Comment:
I think it would be helpful to add a docstring for this code, so the users can understand the difference between this code and `dswx_sar.py`.

Reply:
@seongsujeong `dummpy_dswx_s1.py` code will be removed in the future release. This code is made for interface delivery, but all lines implemented is simply using the fixed thresholds. Anyway, the new codes will be added. Please ignore the errors or missing docstring in `dummpy_dswx_s1.py`